core/api/utils/variants.py

```diff
@@ -10,7 +10,7 @@ def create_or_get_filter_obj(filter_value):
     if core.models.Filter.objects.filter(filter__iexact=filter_value).exists():
         return core.models.Filter.objects.filter(filter__iexact=filter_value).last()
     filter_serializer = core.api.serializers.CreateFilterSerializer(
-        data={"filter": filter_value}
```

Comment:
I'd prefer this in low case, is it possible?

Reply:
Maybe we can apply lowercase to all the fields prior to sending the request